HolyshiftService/PasswordHashService.cs

```diff
@@ -0,0 +1,30 @@
+using Konscious.Security.Cryptography;
+using System;
+using System.Text;
+
+public class PasswordHashService
+{
+    public string GenerateRandomSalt(int size)
+    {
+        byte[] salt = new byte[size];
+        using (var rng = System.Security.Cryptography.RandomNumberGenerator.Create())
```

Comment:
add `using System.Security.Cryptography;` in the beginning of the file. Will make this much shorter and easier to read.